USAGE.md

```diff
@@ -31,6 +32,50 @@ How to test your TMT plans locally? Run a command:
 ```bash
 $ tmt run plan --name <your TMT plan name>
 ```
+
+### Example TMT plan
+How can TMT plan can look like?
+
+```yaml
+summary: TMT/TFT plan for tests on your host
+description: |
+    Run tests on host
+
+discover:
+    how: shell
+    tests:
+    - name: Run your tests on specific host
+      framework: shell
+      test: <run your test suite in a specific directory>
+      duration: 3h
+
+prepare:
+    - name: Install important packages for your tests
+      how: shell
+      script: |
+        # install packages on guest system
+
+    - name: Install packages on machine by ansible
+      how: ansible
+      playbook:
+        - ansible-plan.yml
```

Comment:
Where is this defined/explained?

Reply:
Good point. This is not defined anywhere this is an example. 

The question is if we want to show examples as ansible.

Reply:
Fixed by https://github.com/sclorg/testing-farm-as-github-action/pull/68/commits/c39701a3ec39918bc76c4bddbedcb14eb7e05674